### PR TITLE
add SCRIPT_NAME to all scripts and standardize whiptail-titles

### PR DESCRIPTION
**Pull request by szaimen**
fix point 1 of #1399

### CODE REVIEW

**Comment by szaimen** [comment]
@enoch85 you are welcome to change the names to something more appropriate, if applicable.

File: apps/bitwarden_mailconfig.sh [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 # shellcheck disable=2034,2059
 true
+SCRIPT_NAME="Bitwarden mail-configuration"
```
Comment (enoch85) [inline apps/bitwarden_mailconfig.sh:7]: Bitwarden Mail Configuration

Please follow the same wording everywhere. 

Capital Capital Capital Letter

Thanks!

File: apps/fulltextsearch.sh [hunk 2]
```diff
@@ -5,6 +5,7 @@
 
 # shellcheck disable=2034,2059
 true
+SCRIPT_NAME="Fulltextsearch"
```
Comment (enoch85) [inline apps/fulltextsearch.sh:8]: Full Text Seach

File: apps/onlyoffice_docker.sh [hunk 3]
```diff
@@ -4,6 +4,7 @@
 
 # shellcheck disable=2034,2059
 true
+SCRIPT_NAME="OnlyOffice (docker)"
```
Comment (enoch85) [inline apps/onlyoffice_docker.sh:7]: (Docker)

File: apps/onlyoffice_integrated.sh [hunk 4]
```diff
@@ -4,6 +4,7 @@
 
 # shellcheck disable=2034,2059
 true
+SCRIPT_NAME="OnlyOffice (integrated)"
```
Comment (enoch85) [inline apps/onlyoffice_integrated.sh:7]: (Integrated)

**Comment by enoch85** [comment]
Made some comments, you get the idea. :+1:

**Comment by szaimen** [comment]
> Made some comments, you get the idea. 👍

If you want me to change everything, please also comment on everything you want to have changed. Otherwise I will most likely forget something. 
Easier (at least for me) would be, if you would adjust it yourself 😉
Because I am planning to do the other PRs now.

**Comment by enoch85** [comment]
I suppose you work in GIT?

If not, try it, it's easier. :)

**Comment by szaimen** [comment]
> I suppose you work in GIT?
> 
> If not, try it, it's easier. :)

Yes, I work with a git client but it would be great, if you could do those changes 😉

**Comment by enoch85** [comment]
Haha, sure, will do it when I have time. :)

**Comment by enoch85** [comment]
@szaimen Will push this now.

**Pull request merged**